### ISSUE TITLE
support for Better Minecraft Forge modpack 1.16.5?

### ISSUE THREAD

**Issue by rohankmr414** [cpburnz/minecraft-prometheus-exporter#8]
is it possible to use this modpack with Better Minecraft 1.16.5 forge server -> https://www.curseforge.com/minecraft/modpacks/better-mc-modpack

**Comment by rohankmr414** [comment]
@cpburnz I'm trying to use it with the latest version of bettermc but it doesn't seem to work.

**Comment by cpburnz** [comment]
@rohankmr414 Can you expand upon what specifically is not working? Does Minecraft crash on starting the world? Can you access the exporter locally from the Minecraft server at http://127.0.0.1:19565? Can you not access the exporter from a separate computer on the network (e.g., your computer 192.168.0.111 cannot access the server at http://192.168.0.222:19565)? Can you not access the exporter from a hosted Minecraft server (e.g., http://minecraft.example.com:19565)? Are you accidentally using "https:" instead of "http:"?

In `logs/latest.txt` after start up, shortly after the line:

> [12:26:41] [Server thread/INFO]: Done (11.091s)! For help, type "help"

Do you see a line similar to:

> [12:26:42] [Server thread/INFO]: Listening on 0.0.0.0:19565

**Comment by cpburnz** [comment]
@rohankmr414 I just tested this on the latest version of Better Minecraft, Better MC [FORGE] 1.16.5 v59.5, and it worked for me. I had to try starting Better MC twice because it froze during the first start (unrelated to the Prometheus mod). Once it came up, I shut it down, copied the Prometheus Exporter mod into *mods/*, started the server, waited a couple minutes for it to start up, and then I could access the metrics at "http://127.0.0.1:19565".
